src/libstd/os/raw/uint.md

```diff
@@ -0,0 +1,7 @@
+Equivalent to C's `unsigned int` type.
+
+This type will almost always be [`u16`], but may differ on some esoteric systems. The C standard technically only requires that this type be an unsigned integer with the same size as an [`int`]; some systems define it as a [`u16`], for example.
```

Comment:
Is that first `u16` supposed to be a `u32`?

Reply:
...yes